pkg/apis/sources/v1alpha2/sinkbinding_conversion_test.go

```diff
@@ -18,11 +18,24 @@ package v1alpha2
 
 import (
 	"context"
+	"errors"
 	"testing"
+
+	"knative.dev/pkg/apis"
 )
 
+type dummy struct{}
```

Comment:
nit: Can we move dummy to some test helper? It looks like this can be used for all conversion tests of different resources.

Reply:
Agreed offline that we will do it in another PR, and we will move out the test_helper from apis, otherwise it's in prod code

Reply:
Synced offline. We can do that in follow up PRs